### PR TITLE
feat(plasma-typo): Add tokens for new typo and `Sbermarket` archetype

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[sbermarket_ios-swift--canary.199.3321669717.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/sbermarket_ios-swift--canary.199.3321669717.swift)  
[sbermarket_kotlin--canary.199.3321669717.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/sbermarket_kotlin--canary.199.3321669717.kt)  
[sbermarket_react-native--canary.199.3321669717.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/sbermarket_react-native--canary.199.3321669717.ts)  
[sberprime_ios-swift--canary.199.3321669717.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/sberprime_ios-swift--canary.199.3321669717.swift)  
[sberprime_kotlin--canary.199.3321669717.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/sberprime_kotlin--canary.199.3321669717.kt)  
[sberprime_react-native--canary.199.3321669717.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/sberprime_react-native--canary.199.3321669717.ts)  
[colors--canary.199.3321669717.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.199.3321669717.xml)  
[PlasmaTokensColor--canary.199.3321669717.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.199.3321669717.swift)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.106.0-canary.199.3321669717.0
  npm install @salutejs/plasma-core@1.78.0-canary.199.3321669717.0
  npm install @salutejs/plasma-temple@1.104.0-canary.199.3321669717.0
  npm install @salutejs/plasma-typo@0.25.0-canary.199.3321669717.0
  npm install @salutejs/plasma-ui@1.138.0-canary.199.3321669717.0
  npm install @salutejs/plasma-web@1.137.0-canary.199.3321669717.0
  npm install @salutejs/plasma-cy-utils@0.31.0-canary.199.3321669717.0
  # or 
  yarn add @salutejs/plasma-b2c@1.106.0-canary.199.3321669717.0
  yarn add @salutejs/plasma-core@1.78.0-canary.199.3321669717.0
  yarn add @salutejs/plasma-temple@1.104.0-canary.199.3321669717.0
  yarn add @salutejs/plasma-typo@0.25.0-canary.199.3321669717.0
  yarn add @salutejs/plasma-ui@1.138.0-canary.199.3321669717.0
  yarn add @salutejs/plasma-web@1.137.0-canary.199.3321669717.0
  yarn add @salutejs/plasma-cy-utils@0.31.0-canary.199.3321669717.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-199/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-199/
  ui storybook: http://plasma.sberdevices.ru/pr-199/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-199/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-199/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-199/temple-storybook/

File: packages/plasma-typo/src/archetypes/mage.ts [hunk 1]
```diff
@@ -520,9 +520,191 @@ export const baseTypoL = {
...
+export const typoCommonProps = (displayFontFamily: string, textFontFamily: string) => ({
+    'dspl-l': {
+        'font-family': displayFontFamily,
+        'letter-spacing': 'normal',
+        'font-style': 'normal',
+    },
+    'dspl-l-bold': {
+        'font-family': displayFontFamily,
+        'letter-spacing': 'normal',
+        'font-style': 'normal',
```
Comment (Yeti-or) [inline packages/plasma-typo/src/archetypes/mage.ts:532]: А жирность у нас тут не контролируется ?

File: packages/plasma-typo/src/archetypes/mage.ts [hunk 2]
```diff
@@ -520,9 +520,191 @@ export const baseTypoL = {
...
+    },
+    'text-l': {
+        'font-family': textFontFamily,
+        'letter-spacing': '-0.019em',
+        'font-style': 'normal',
+    },
+    'text-l-bold': {
+        'font-family': textFontFamily,
+        'letter-spacing': '-0.019em',
```
Comment (Yeti-or) [inline packages/plasma-typo/src/archetypes/mage.ts:661]: Em осознанное решение да ?

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-typo/src/archetypes/mage.ts (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-typo/src/archetypes/mage.ts:532]: не, жирность контролится в сущности выше

File: packages/plasma-typo/src/archetypes/mage.ts (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-typo/src/archetypes/mage.ts:661]: да. В макете фигмы указаны проценты, веб не умеет в относительные величины, поэтому с помощью вот этой формулы https://help.figma.com/hc/en-us/articles/360039956634-Explore-text-properties#letter-spacing конвертировал в em

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-199/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-199/
  ui storybook: http://plasma.sberdevices.ru/pr-199/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-199/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-199/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-199/temple-storybook/

**Pull request merged**